pages/integrations/sso/azure_ad.md.erb

```diff
@@ -92,8 +92,7 @@ Then go to your [Azure Admin Console](https://portal.azure.com/) and select the
     1. Toggle _Enabled_ to _No_ and click _Save_
 1. Customise the User mappings:
     1. Expand _Mappings_, then click _Provision Azure Active Directory Users_
-    1. Click the first row of the _Attribute Mappings_ table, _userPrincipalName_ and change the _Source attribute_ to `mail` then click _OK_
-    1. Delete all remaining mappings except for the four listed in the following screenshot:
+    1. Delete all mappings except for the four listed in the following screenshot:
```

Comment:
Could we list these in text? Relying on the image makes it hard for folks using screen readers, and also if they change their UI things get confusing 😄